src/persistence/settings.cpp

```diff
@@ -235,7 +235,7 @@ void Settings::loadGlobal()
             else
                 style = "None";
         }
-		avatarsPath = s.value("avatarsPath",QDir::homePath()).toString();
+        avatarsPath = s.value("avatarsPath",QDir::homePath()).toString();
```

Comment:
I think here you should use `setAvatarPath` function method which would validate that the path exists and is writeable, I see big potential for breaking stuff here.